addons/l10n_pe/models/res_partner.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_pe_edi_district = fields.Many2one(
```

Comment:
`l10n_pe_edi_district` -> `l10n_pe_district` this is a peruvian field not a peruvian field for EDI.

Reply:
Done